framework/include/utils/MooseADWrapper.h

```diff
@@ -15,8 +15,12 @@
 #include "RankThreeTensor.h"
 #include "RankFourTensor.h"
 
+#include "libmesh/dense_vector.h"
 #include "libmesh/vector_value.h"
 #include "libmesh/tensor_value.h"
+#include "libmesh/dense_matrix.h"
+#include "libmesh/dense_matrix_base.h"
+#include "libmesh/dense_matrix_impl.h"
```

Comment:
I don't believe this include should actually be needed, either here or in the .C file. It doesn't appear to me that we instantiate any of the methods defined in the `impl` file. I could have missed something though

Reply:
removed.